autotest/gcore/transformer.py

```diff
@@ -243,6 +243,27 @@ def test_transformer_5():
 
     tr = None
 
+    # Test RPC_DEM_SRS by adding vertical component egm 96 geoid
```

Comment:
could you move the below lines in a separate test_XXXXX method() to avoid the current one to go out of control ?